dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -502,6 +532,154 @@ public void testWorkflowVersionIncorrectPath() {
 
     }
 
+    /**
+     * Tests that parsing a GitLab URL works
+     */
+    @Test
+    public void testGitLabUrlRegexParsing() {
```

Comment:
These, the BitBucket, and GitHub tests seem like they can all be unit tests. There's overhead in running integration tests, so it would be better if they can be unit tests. But if they can't be unit tests as they are, it's fine to leave them.

Reply:
made them unit tests in SourceCodeRepoFactoryTest